MOBILE_CLIENT/COMMON/_MEDIUM/INSECURE_DYNAMICALLY_LINKED_LIBRARY/description.md

```diff
@@ -0,0 +1,6 @@
+Dynamicaly loading libraries if not handled properly gives room for a range of potential vulnerabilities including:
```

Comment:
Ping to add the source code samples and improve the text.

Reply:
Added. [480eb64](https://github.com/Ostorlab/KB/pull/40/commits/480eb64fc4e73f1b4b754a5ec71f0cf45ca0f301) [2b4948d](https://github.com/Ostorlab/KB/pull/40/commits/2b4948d69937faa91ff10a3b0c433145e7619445)